azure-pipelines.yml

```diff
@@ -0,0 +1,222 @@
+variables:
+  ITKGitTag: v5.0rc01
+  CMakeBuildType: MinSizeRel
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+
+jobs:
+
+- job: 'Test'
+  displayName: "Build and test"
+  timeoutInMinutes: 0
+  cancelTimeoutInMinutes: 300
+
+  strategy:
+    matrix:
+      Linux:
+        imageName: 'ubuntu-16.04'
+        cCompiler: gcc
+        cxxCompiler: g++
+        compilerInitialization: ''
+      macOS:
+        imageName: 'macos-10.13'
+        cCompiler: clang
+        cxxCompiler: clang++
+        compilerInitialization: ''
+      Windows:
+        imageName: 'vs2017-win2016'
+        cCompiler: cl.exe
+        cxxCompiler: cl.exe
+        compilerInitialization: 'call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvars64.bat"'
+
+  pool:
+    vmImage: $(imageName)
+
+  steps:
+  - bash: |
+      set -x
+      if [ -n "$(System.PullRequest.SourceCommitId)" ]; then
+        git checkout $(System.PullRequest.SourceCommitId)
+      fi
+    displayName: 'Checkout pull request HEAD'
+
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.7'
+      architecture: 'x64'
+
+  - script: |
+      python -m pip install --upgrade pip
+      python -m pip install --upgrade setuptools
+      python -m pip install scikit-ci-addons
+      python -m pip install ninja
+    displayName: 'Install build dependencies'
+
+  - script: |
+      git clone --depth 5 --branch $(ITKGitTag) https://github.com/InsightSoftwareConsortium/ITK.git
+    workingDirectory: $(Agent.BuildDirectory)
+    displayName: 'Download ITK'
+
+  - script: |
+      mkdir ITK-build
+      cd ITK-build
+      $(compilerInitialization)
+      cmake -DCMAKE_C_COMPILER:FILEPATH="$(cCompiler)" -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_CXX_COMPILER="$(cxxCompiler)" -DCMAKE_BUILD_TYPE:STRING=$(CMakeBuildType) -DBUILD_TESTING:BOOL=OFF -GNinja ../ITK
+      ninja
+    workingDirectory: $(Agent.BuildDirectory)
+    displayName: 'Build ITK'
+
+  - script: |
+      curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITK/dashboard/itk_common.cmake -O
+    displayName: 'Fetch CTest driver script'
+
+  - script: |
+      python -m pip install cookiecutter
+      cd /
+      python -m cookiecutter --no-input /ITKModuleTemplateCookieCutter
```

Comment:
Here we want `/ITKModuleTemplateCookieCutter` -> `--output-dir $(Agent.BuildDirectory)/ITKModuleTemplate $(Agent.SourcesDirectory)`

Reply:
Done in a587ac1.